src/app/declarations/models/login-request.ts

```diff
@@ -0,0 +1,5 @@
+/* tslint:disable */
+export interface LoginRequest {
+  username: string;
+  admin: boolean;
```

Comment:
A to chyba nieaktualne, bo `admin` już nie ma na masterze

Reply:
Po pobraniu nowego api, stare się zastąpi

Reply:
Aaaa, to są te one te. To powinno trafiać na repo?

Reply:
moim zdaniem tak - można wtedy uruchomić aplikację bez tego całego pobierania submodułów i generacji

no ale jeśli się uprzecie, to można to uznać za coś w stylu javowych generowanych interfejsów

Reply:
> no ale jeśli się uprzecie, to można to uznać za coś w stylu javowych generowanych interfejsów

Ja jestem w liczbie pojedynczej, ale dla mnie to brzmi spoko

Reply:
wybacz, ale jakoś założyłem, że ktoś jeszcze może zechce wyrazić swoje zdanie xd stąd liczba mnoga

Reply:
No spoko, na razie masz głos jednej osoby. Jak chcesz więcej, to można otagować resztę albo zapytać na Slacku

Reply:
To ja mogę zechcieć wyrazić. Popieram podejście @szbrnl, choćby z powodu łatwiejszego setupu aplikacji. Przy okazji - czy istnieje jakaś oficjalna konwencja co do repo trafiać powinno a co nie?

Reply:
https://softwareengineering.stackexchange.com/questions/192113/do-i-check-generated-code-in-to-source-control-or-not

https://hackernoon.com/what-should-be-in-version-control-d5f16e9a2bf2

https://www.reddit.com/r/git/comments/23i7ln/generated_source_code_and_source_control/

Reply:
a z drugiej strony masz automatycznie generowany package-lock.json, który podobno należy commitować

Reply:
Są reguły, są i wyjątki od nich.

Reply:
Poza tym, zależy jak zdecydujemy w kwestii dwóch zgodnych masterów. Jeśli niekoniecznie musi być zawsze najnowsze api, to trzymanie wygenerowanych deklaracji już nie będzie chyba takim strasznym przestępstwem xd

Reply:
Akurat jak dla mnie trochę odwrotnie. Skoro stan tego repo w dowolnym momencie odzwierciedla konkretny stan `papaya-interface`, to ta generacja może się zawsze odbyć jednoznacznie

Reply:
No cóż, w każdym razie już jakiś czas temu usunąłem te pliki i dodałem je do gitignora

Reply:
Zauważyłam